Solutions/Infoblox Cloud Data Connector/Analytic Rules/Infoblox-HighNumberOfHighThreatLevelQueriesDetected.yaml

```diff
@@ -0,0 +1,42 @@
+id: 57113ad7-7dd6-4150-84d8-252e162aaf4a
+name: Infoblox - High Number of High Threat Level Queries Detected
+description: |
+  'This creates an incident in the event a single host generates a high number of high threat level queries.'
```

Comment:
Could you please add any reference to product logging documentation.. e.g. i found these https://docs.infoblox.com/display/BloxOneThreatDefense/Response+Log+File+Format

Reply:
@ashwin-patil, Since the queries are pretty broad I'm not sure any documentation we have makes sense that isn't already on the data connector and (future) marketplace offer. Those are heavily documented + with links. Is it preferable to reiterate here?